RadClass/RadClass.py

```diff
@@ -179,7 +179,7 @@ def iterate(self):
         while running:
             # print status at set intervals
             if np.where(self.processor.timestamps == self.working_time)[0][0] % log_interval == 0:
-                bar.update(round((self.working_time - self.processor.timestamps[0]) * inverse_dt, 4)*100)
+                bar.update(round((self.working_time - self.processor.timestamps[0]) * inverse_dt * 100, 4))
```

Comment:
I moved this multiplication to the inside of `round()` because it was messing with the number of digits printed.